composer.json

```diff
@@ -17,6 +17,7 @@
         "drupal/gin": "^3.0@alpha",
         "drupal/gin_login": "^2.0.3",
         "drupal/gin_toolbar": "^1.0@beta",
+        "drupal/entity_browser": "^2.9",
```

Comment:
Do we need this here?

Reply:
Yes, it seems so! I think it only patches if the require is in the same composer.json

I'm still not 100% sure though.

Here's a test run without the require which has failures: 
https://github.com/localgovdrupal/localgov/actions/runs/6810487415

Here's the one with the require, which passed: 
https://github.com/localgovdrupal/localgov/actions/runs/6810600296